server/args.py

```diff
@@ -0,0 +1,18 @@
+# args.py
+import argparse
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "--sqlite-file-path",
+        default="pastes.db",
+        help="The path to the SQLite database file.",
```

Comment:
say what the default value is in the help text

Reply:
Changed it